lib/ace/mode/batchfile_highlight_rules.js

```diff
@@ -46,7 +46,7 @@ var BatchFileHighlightRules = function() {
 
     this.$rules = { start: 
        [ { token: 'keyword.command.dosbatch',
-           regex: '\\b(?:append|assoc|at|attrib|break|cacls|cd|chcp|chdir|chkdsk|chkntfs|cls|cmd|color|comp|compact|convert|copy|date|del|dir|diskcomp|diskcopy|doskey|echo|endlocal|erase|fc|find|findstr|format|ftype|graftabl|help|keyb|label|md|mkdir|mode|more|move|path|pause|popd|print|prompt|pushd|rd|recover|ren|rename|replace|restore|rmdir|set|setlocal|shift|sort|start|subst|time|title|tree|type|ver|verify|vol|xcopy)\\b',
+           regex: '\\b(?:append|arp|assoc|at|attrib|autofail|backup|basica|bcdedit|bootcfg|break|cacls|cd|cdburn|certreq|certutil|change|chcp|chdir|chglogon|chgport|chgusr|chkdsk|chkntfs|choice|cipher|cleanmgr|cls|cmd|cmdkey|color|command.com|comp|compact|%comspec%|control|convert|copy|cprofile|cscript|csvde|ctty|date|debug|defrag|del|deltree|devcon|dhcploc|diantz|dir|diskcomp|diskcopy|diskpart|doskey|dpath|driverquery|dsadd|dsget|dsmod|dsmove|dsquery|dsrm|dvdburn|echo|edit|edlin|endlocal|epal|erase|eventcreate|eventquery.vbs|eventtriggers|expand|explorer|extract|exe2bin|fastopen|fc|fdisk|filever|find|findramd|findstr|for|forcedos|forfiles|format|fsutil|ftp|ftype|getmac|gettype|gpresult|graftabl|gwbasic|help|icacls|iexpress|if|ifconfig|inuse|ipconfig|jt|keyb|label|lfnfor|lh|loadhigh|lock|logman|logoff|makecab|md|mem|mkdir|mklink|mode|more|mountvol|move|msd|msg|msiexec|msinfo32|nbtstat|net|netdom|netsh|netstat|nltest|notepad|nslookup|ntbackup|ocsetup|odbcconf|openfiles|path|pathping|pause|pentnt|ping|ping6|pkgmgr|popd|powercfg|print|printui.exe|prncfg.vbs|prndrvr.vbs|prnjobs.vbs|prnmngr.vbs|prnport.vbs|prnqctl.vbs|prompt|pubprn.vbs|pushd|qappsrv|qbasic|qchain|qfarm|qprocess|qserver|query|quser|qwinsta|rasdial|rasphone|rd|readline|recimg|recover|reg|regedit|regedt32|regini|regsvr32|rem|ren|rename|replace|reset session|restore|rmdir|robocopy|route|runas|rundll|rundll32|sc|schtasks|secedit|set|setlocal|sfc|shadow|shell|shift|shutdown|sort|start|subinacl|subst|systeminfo|taskkill|tasklist|time|title|tracert|tracert6|tracerte|tree|truename|tscon|tsdiscon|tskill|tsprof|tsshutdn|type|tzchange|unlock|ver|verify|vhdmount|vol|winmsd|winmsdp|winsat|wmic|wscript|wupdmgr|xcacls|xcopy)\\b',
```

Comment:
`regedt32`? This seems like a fairly open-ended list? Should we have all of these?